components/kyma-environment-broker/internal/process/provisioning/create_runtime.go

```diff
@@ -112,6 +112,12 @@ func (s *CreateRuntimeStep) Run(operation internal.ProvisioningOperation, log lo
 		return operation, 1 * time.Minute, nil
 	}
 	instance.RuntimeID = *provisionerResponse.RuntimeID
+	// Save provider region in instance ProvivisiongParameters so that all instances store it regardless of ServicePlan, and whether region parameter was input or not
+	pp.Parameters.Region = &requestInput.ClusterConfig.GardenerConfig.Region
```

Comment:
This region is a value provided by a OSB API client. When I'm creating an instance with region="europe" I do not expect the GET instance response with region="westeurope". it must be the same

Reply:
We're adding a new field so on the `/runtimes` response there will be the region provided by user and the actual region used in Gardener

Reply:
Yes, I found the specific problem with trial plan. Both platformRegion and params.Region can contain the pseudo region value like "europe", and is mapped to provider region.
For getting a clear view on the runtimes we need to know the provider region exactly, so I will instead introduce a new field "ProviderRegion" in instance and store the calculated or provided provider specific region there.